docs/nx-cloud/concepts/cache-security.md

```diff
@@ -32,9 +32,14 @@ You can strengthen your workspace security further by revoking all access to the
 
 While you can [specify a token](/ci/recipes/security/access-tokens) with the `nxCloudAccessToken` property in `nx.json`, this is visible to anyone who can view your codebase. A read-write token would give someone who may not even have permission to create a PR the access to add entries to the remote cache, which would then be used on other developer's machines and in CI. We recommend you restrict CI Access Tokens to CI use only and rely on [personal access tokens](/ci/recipes/security/personal-access-tokens) for local development instead.
 
-### Use a Read-Write Token in CI
+### Use Scoped Tokens in CI
```

Comment:
More for the paragraph above, 

`A read-write token would give someone who may not even have permission to create a PR the access to add entries to the remote cache, which would then be used on other developer's machines and in CI.` is a little wordy and hard to read